cherab/core/beam/node.pyx

```diff
@@ -232,7 +235,24 @@ cdef class Beam(Node):
 
     cpdef Vector3D direction(self, double x, double y, double z):
         """
```

Comment:
Use `r"""` and get rid of double backslashes in math formulae.

Reply:
Done.